src/store/modules/layers.store.js

```diff
@@ -144,6 +144,18 @@ const getters = {
             (layer) => layer.visible && layer.type === LayerTypes.KML && !layer.isExternal
         ) ?? null,
 
+    /**
+     * Get index of current KML layer selected for drawing.
+     *
+     * That is the KML layer that will be used when the drawing mode is opened.
+     *
+     * When no KML layer is in active layers then -1 is returned.
```

Comment:
That is why you've added this getter, but it can work for other type of layers too. You might want to reflect that in the comment and have something more general as a description (and maybe end by saying this is mainly used to get the index of the active KML layer)